e2e/angular-core/src/angular-linting.test.ts

```diff
@@ -64,7 +64,6 @@ describe('Angular Package', () => {
       expect(appLintStdOut).toContain(
         `The selector should start with one of these prefixes`
       );
-      expect(appLintStdOut).toContain(`7:18`);
```

Comment:
Why is this assertion removed?

Reply:
I may be wrong, but I believe this is checking the exact line number that the error occurred on, or perhaps the error number? Either way, this part of the error message changed, but the error itself (the text above) did not. If this is subject to change across versions of typescript-eslint then I'd rather not rely on it, as it does not provide much value.